internal/actioninfo/actioninfo.go

```diff
@@ -1,11 +1,31 @@
 package actioninfo
 
 import (
-	...
+	"errors"
+	"fmt"
 )
 
 // создайте интерфейс DataParser
-...
+type DataParser interface {
+	Parse(string) error
+	ActionInfo() (string, error)
+}
 
 // создайте функцию Info()
-...
+func Info(dataset []string, dp DataParser) {
+	for _, data := range dataset {
+		if err := dp.Parse(data); err != nil {
+			err = errors.New("parsing error")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
			err = fmt.Errorf("parsing error: %w", err)
```